quickstart/docker/image/run-controller.sh

```diff
@@ -1,21 +1,19 @@
 #!/bin/bash
-
 ziti_controller_cfg="${ZITI_HOME}/ziti-edge-controller.yaml"
 
-export ZITI_CONTROLLER_RAWNAME=ziti-controller
-export ZITI_EDGE_CONTROLLER_RAWNAME=ziti-edge-controller
+export ZITI_CTRL_EDGE_ADVERTISED_ADDRESS=ziti-edge-controller
 
 . "${ZITI_SCRIPTS}/ziti-cli-functions.sh"
 
-# access-control.init is created by ziti-controller-init-container after initial controller startup
+echo "ZITI_CTRL_EDGE_ADVERTISED_ADDRESS is ${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}"
 if [ ! -f "${ZITI_HOME}/access-control.init" ]; then
-  generateEnvFile
+  setupEnvironment
+  persistEnvironmentValues
 fi
-. "${ZITI_HOME}/ziti.env"
```

Comment:
why remove quotes? if there's spaces in ziti home this becomes a problem. hope that's not intentional?